[2024-03-15 18:50:04,264] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-15T18:49:00+00:00 [queued]>
[2024-03-15 18:50:04,286] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-15T18:49:00+00:00 [queued]>
[2024-03-15 18:50:04,287] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 18:50:04,287] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2024-03-15 18:50:04,287] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 18:50:04,299] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2024-03-15T18:49:00+00:00
[2024-03-15 18:50:04,307] {standard_task_runner.py:52} INFO - Started process 50426 to run task
[2024-03-15 18:50:04,310] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2024-03-15T18:49:00+00:00', '--job-id', '3501', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpys00yp4w', '--error-file', '/tmp/tmp064hf4vx']
[2024-03-15 18:50:04,311] {standard_task_runner.py:77} INFO - Job 3501: Subtask retrieve_raw_data
[2024-03-15 18:50:04,363] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2024-03-15T18:49:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-15 18:50:04,437] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T18:49:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T18:49:00+00:00
[2024-03-15 18:50:04,904] {python.py:151} INFO - Done. Returned value was: None
[2024-03-15 18:50:04,918] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20240315T184900, start_date=20240315T185004, end_date=20240315T185004
[2024-03-15 18:50:04,961] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-03-15 18:50:05,005] {local_task_job.py:151} INFO - Task exited with return code 0
